azavea/fb-gender-survey-dashboard
# Mark Gender Chart as Percentage

## Overview

Adds "What is your gender" as question text that is recognized as a percentage chart type. 

Updates the data to the newest version. 

Connects #88 

### Demo

<img width="1190" alt="Screen Shot 2021-02-02 at 1 16 37 PM" src="https://user-images.githubusercontent.com/21046714/106644668-61aaa480-6559-11eb-9c3d-7079dbb459b3.png">

## Testing Instructions

 * In the Netlify preview, select several locations.
 * Skim over the questions and confirm that the punctuation generally appears to be correct (for example, '* main*', with a space after the first asterisk, should not appear in the updated data). 
 * Select several questions, including 'What is your gender?', and navigate to the charts page.
 * Confirm that the percentage symbols appear with charts where it makes sense, and that 'What is your gender?' is treated as a percent.
